files/ja/web/guide/user_input_methods/index.html

```diff
@@ -19,11 +19,11 @@
 
 <h2 id="User_input_and_controls_workflow" name="User_input_and_controls_workflow">ユーザー入力とコントロールのワークフロー</h2>
 
-<p>The following diagram illustrates the typical workflow for implementing user input mechanisms:</p>
+<p>次の図式は、ユーザー入力の仕組みを実装するための典型的なワークフローを説明しています。</p>
 
 <p style="text-align: center;"><img alt="" src="https://mdn.mozillademos.org/files/8085/user-input-and-controls.png" style="height: 86px; width: 303px;"></p>
 
-<p>First of all, you need to decide which input mechanisms you want to cover in your application out of mouse, keyboard, finger touch and so on. Once you decided the input mechanisms, you can control them using tools offered by the web platform or JavaScript libraries.</p>
+<p>最初に、マウス、キーボード、指でのタッチなどから、アプリケーションで対象としたい入力の仕組みをどれにするかを決める必要があります。入力の仕組みを決めたのであれば、ウェブプラットフォームや JavaScript ライブラリによって提供されているツールを使い、制御することができます。</p>
```

Comment:
```suggestion
<p>最初に、マウス、キーボード、指でのタッチなどから、アプリケーションで対象としたい入力の仕組みをどれにするかを決める必要があります。入力の仕組みを決めたのであれば、ウェブプラットフォームや JavaScript ライブラリーによって提供されているツールを使い、制御することができます。</p>
```
ライブラリ→ライブラリーの修正をお願いします。

Reply:
@mfuji09 レビューありがとうございます！こちら取り込みました！